.editorconfig

```diff
@@ -0,0 +1,21 @@
+# http://editorconfig.org
+
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+insert_final_newline = true
+charset = utf-8
+end_of_line = lf
+
+[*.bat]
+indent_style = tab
+end_of_line = crlf
```

Comment:
😅 
I actually haven't touched or tested the .editorconfig
There are a couple of bat files so that windows users can contribute, but I haven't test those

Reply:
do you think we should remove this file?

Reply:
The `.editorconfig` file is fine I just missed the `docs/make.bat`. Ignore my comment.